### PR TITLE
feat(mtd): allow updated url on resume

### PR DESCRIPTION
**Pull request by continuata**
addressing issue #83

### CODE REVIEW

**Comment by continuata** [comment]
resume currently doesn't work for some reason ... passing a new url with the download just performs a new download, omitting the URL doesn't resume either

File: src/downloadMTD.js [hunk 1]
```diff
@@ -12,9 +14,10 @@ const metaUpdate = require('./metaUpdate')
 const create = require('reactive-storage').create
 const Immutable = require('immutable')
 
-module.exports = (ob, fd) => {
+module.exports = (ob, fd, options) => {
   const offsets = create(Immutable.List([]))
   const loadedMETA = metaLoad(fd)
+    .map(meta => _.assign({}, options, meta))
```
Comment (tusharmath) [inline src/downloadMTD.js]: shouldn't `meta` be first?

```js
_.assign({}, meta, options)
```

Comment (continuata) [inline src/downloadMTD.js]: maybe ... I just copied what you wrote in the issue comment :-)

However, this still doesn't fix the issue

**Comment by tusharmath** [comment]
closing this in lieu of #96

**Pull request closed without merging**